src/Helper/Log/GoogleApiClient.php

```diff
@@ -212,7 +212,7 @@ public function getToken(): ?string {
     }
     catch (ExceptionInterface $e) {
       $this->output->comment('An error occurred accessing the auth token from Google API endpoint.\n' . $e->getMessage());
-      exit;
+      return NULL;
```

Comment:
You don't actually have to specify `return NULL`. Just `return` does the same thing.

```suggestion
      return;
```

Reply:
Its throwing error https://github.com/acquia/orca/actions/runs/5296402120/jobs/9587488078#step:5:1932